src/App.tsx

```diff
@@ -1,24 +1,152 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import {
+  FC, useCallback,
+  useEffect,
+  useMemo,
+  useState,
+} from 'react';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { ErrorType } from './types/Error';
+import { TaskStatus } from './types/Sort';
+import { deleteTodo, getTodos, postTodo } from './api/todos';
+import { AddTodo } from './components/AddTodo';
+import { TodoList } from './components/TodoList';
+import { ErrorNotification } from './components/ErrorNotification';
+import { Filter } from './components/Filter';
+import { getFilteredTodos } from './utils/helpers';
+import { DEFAULT_TASK_ID, USER_ID } from './utils/constants';
 
-const USER_ID = 0;
+export const App: FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [error, setError] = useState<ErrorType>(ErrorType.NONE);
+  const [sortType, setSortType] = useState<TaskStatus>(TaskStatus.ALL);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [isDisabled, setIsDisabled] = useState<boolean>(false);
+  const [loadTodoById, setLoadTodoById] = useState([DEFAULT_TASK_ID]);
+
+  const clearErrorMessage = () => {
+    setTimeout(() => setError(ErrorType.NONE), 3000);
+  };
+
+  const fetchTodos = useCallback(async () => {
+    try {
+      const getData = await getTodos(USER_ID);
+
+      setTodos(getData);
+    } catch {
+      setError(ErrorType.LOAD);
+      clearErrorMessage();
+    }
+  }, []);
```

Comment:
![image](https://user-images.githubusercontent.com/102293862/233034977-2cff35f0-5a03-4d03-b9d8-7d9700befa43.png)
Rather than wrapping it with useCallback just move this function inside the useEffect.
You can read more about usage of useCallback here: https://react.dev/reference/react/useCallback